src/main/webapp/resources/js/pages/analysis/components/AnalysisDelete.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useContext } from "react";
+import { Button, Checkbox, Alert, Popconfirm, Row } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { showNotification } from "../../../modules/notifications";
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+import { deleteAnalysis } from "../../../apis/analysis/analysis";
+
+export default function AnalysisDelete() {
+  const { analysisContext } = useContext(AnalysisContext);
+  const [deleteConfirm, setDeleteConfirm] = useState(false);
+
+  function onChange(e) {
+    // Set local state for deleteConfirm
+    if (e.target.checked == true) {
+      setDeleteConfirm(true);
+    } else {
+      setDeleteConfirm(false);
+    }
+  }
+
+  function handleDeleteConfirm() {
+    if (deleteConfirm) {
+      deleteAnalysis(analysisContext.analysis.identifier).then(res =>
+        showNotification({ text: res.result })
+      );
+
+      window.setTimeout(function() {
+        window.location.replace(window.TL.BASE_URL);
+      }, 3500);
+    }
+  }
+
+  return (
+    <>
+      <h2 style={{ fontWeight: "bold" }}>
+        {getI18N("analysis.tab.delete-analysis")}
+      </h2>
+      <strong className="spaced-top__sm">
```

Comment:
Wrap the message in `strong` (or use the ant.design Typography `Text` with `strong` attribute, not the entire `Alert`

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a